lib/display_text.rb

```diff
@@ -0,0 +1,5 @@
+module DisplayText
```

Comment:
I like how easy it is to understand this module because of the name, however, `DisplayText.display_text` feels redundant. When I want to invoke a class/module that represents an action, I sometimes use the method `#call`, which is sort of an interesting method in Ruby.

https://ruby-doc.org/core-2.5.3/Method.html#method-i-call

Reply:
commit c5d90ce577f684ee0097c61a7bef8d940054fd1b